File: src/backend/SOVVF/Classi/Geo/Poligonale.cs
using System.Collections.Generic;

namespace Modello.Classi.Geo
{
    /// <summary>
    ///   Modella una poligonale sul globo
    /// </summary>
    public class Poligonale : Geolocalizzazione
    {
        /// <summary>
        ///   Rappresenta la lista dei vertigi della poligonale
        /// </summary>
        /// <remarks>
        ///   Non è richiesto che il primo e l'ultimo vertice corrispondano, chiudendo così la poligonale.
        /// </remarks>
        public IList<Punto> Vertici { get; set; }
    }
}

File: src/frontend/prototipi/cruscottoIntegratoSB/src/app/layout/form-chiamata/shared/classes/geo/poligonale.ts
import { Geolocalizzazione } from "./geolocalizzazione";
import { Punto } from "./punto";

export class Poligonale extends Geolocalizzazione {

    public vertici: Punto[];

    baricentro(): Punto {
        //TODO : Implementare
        return new Punto;
    }

    constructor() {
        super();
    }

}